harness/byteConversionValues.js

```diff
@@ -51,7 +51,22 @@ var byteConversionValues = {
     -4294967296, // - ( 2 ** 32 )
     Infinity,
     -Infinity,
-    0
+    0,
+    2049,                         // an integer which rounds down under ties-to-even when cast to float16
+    2051,                         // an integer which rounds up under ties-to-even when cast to float16
+    0.00006103515625,             // smallest normal float16
+    0.00006097555160522461,       // largest subnormal float16
+    5.960464477539063e-8,         // smallest float16
+    2.9802322387695312e-8,        // largest double which rounds to 0 when cast to float16
```

Comment:
the eslint "no precision loss" rule complains about this value, is that a concern?